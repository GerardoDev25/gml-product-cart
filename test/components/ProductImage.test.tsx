import React from 'react';
import rendered from 'react-test-renderer';
import {ProductImage, ProductCard } from '../../src/components';
import { product2 } from './data/product';

describe('test in productTitle', () => {
  test('should to show the component correctly with the img costumed', () => {
    const wrapper = rendered.create(
      <ProductImage img='custmo.img.png' className="custome-class" />
    );

    expect(wrapper.toJSON()).toMatchSnapshot();
  });

  test('should to show the component with the product img', () => {
    const wrapper = rendered.create(
      <ProductCard product={product2}>{() => <ProductImage />}</ProductCard>
    );

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
