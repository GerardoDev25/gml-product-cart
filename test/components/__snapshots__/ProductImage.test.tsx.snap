// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test in productTitle should to show the component correctly with the img costumed 1`] = `
<img
  alt="Product img"
  className="productImg custome-class"
  src="custmo.img.png"
/>
`;

exports[`test in productTitle should to show the component with the product img 1`] = `
<div
  className="productCard undefined"
>
  <img
    alt="Product img"
    className="productImg undefined"
    src="./coffee-mug.png"
  />
</div>
`;
